frontend/src/pages/IntegrationAuthCallback/IntegrationAuthCallbackPage.tsx

```diff
@@ -708,12 +709,12 @@ const IntegrationAuthCallbackPage = () => {
 					loading: false,
 					currentProject: undefined,
 					allProjects: [],
-					currentWorkspace: workspaceId
-						? { id: workspaceId, name: '' }
-						: {
-								id: currentWorkspaceId,
-								name: '',
-						  },
+					currentWorkspace: {
+						id: workspaceId ? workspaceId : currentWorkspaceId,
```

Comment:
Just wanted to raise for awareness: think this is equivalent to `workspaceId ?? currentWorkspaceId`, which I feel is cleaner but not too opinionated

Reply:
See you are doing this below